src/Apps/Article/Components/ArticleVideo.tsx

```diff
@@ -178,9 +179,12 @@ const ArticleVideo: FC<ArticleVideoProps> = ({ article }) => {
               <Text variant="lg-display">
                 More in{" "}
                 {article.seriesArticle ? (
-                  <RouterLink to={article.seriesArticle.href}>
+                  <Link
+                    as={RouterLink}
+                    href={article.seriesArticle.href || undefined}
```

Comment:
This doesn't make sense that it'd work since by casting with `as` we'd accept props for `RouterLink` so `href` should be `to`